config/initializers/assets.rb

```diff
@@ -0,0 +1,2 @@
+# This asset is in sul_chrome (which is deprecated. See https://github.com/sul-dlss/sul_chrome/issues/1)
+Rails.application.config.assets.precompile += %w(footer-stanford-logo.png)
```

Comment:
Did this git brought in with the sul_chrome import and can now be removed (or at least the comment removed)?